ext/opentelemetry-ext-grpc/tests/test_server_interceptor.py

```diff
@@ -49,6 +53,43 @@ def service(self, handler_call_details):
 
 
 class TestOpenTelemetryServerInterceptor(TestBase):
```

Comment:
One other thing, you should probably add a test to make sure `uninstrument()` works properly

Reply:
uninstrument() is implicitly tested when uninstrument is called. Should this not be working every time grpc.server is called wrapper_fn would also be called. And as a result proceeding tests should fail.